picamera2/encoders/encoder.py

```diff
@@ -64,7 +64,11 @@ def output(self):
 
     @output.setter
     def output(self, value):
-        if not isinstance(value, Output):
+        if isinstance(value, list):
+            for out in value:
+                if not isinstance(out, Output):
+                    raise RuntimeError("Must pass Output")
+        elif not isinstance(value, Output):
```

Comment:
Just a small thing, I wonder if this code (and all the repeats of `isintance(..., list)`) would look tidier if we started this method with
```
    if not isinstance(value, list):
        value = [value]
```
so that we can then assume we always have a list? Perhaps the "getter" could "de-listify" its return value if there's only one thing in the list.

But don't change it unless you think it actually looks more readable!

Reply:
Good idea, just changed the code to do this